src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleEqualityTests.cs

```diff
@@ -1204,10 +1204,10 @@ static void Main()
             comp.VerifyDiagnostics(
                 // (6,30): error CS8315: Operator '==' is ambiguous on operands 'default' and 'default'
                 //         System.Console.Write((default, default) == (default, default));
-                Diagnostic(ErrorCode.ERR_AmbigBinaryOpsOnDefault, "(default, default) == (default, default)").WithArguments("==").WithLocation(6, 30),
+                Diagnostic(ErrorCode.ERR_AmbigBinaryOpsOnTypelessExpression, "(default, default) == (default, default)").WithArguments("==", "default", "default").WithLocation(6, 30),
                 // (6,30): error CS8315: Operator '==' is ambiguous on operands 'default' and 'default'
                 //         System.Console.Write((default, default) == (default, default));
-                Diagnostic(ErrorCode.ERR_AmbigBinaryOpsOnDefault, "(default, default) == (default, default)").WithArguments("==").WithLocation(6, 30),
+                Diagnostic(ErrorCode.ERR_AmbigBinaryOpsOnTypelessExpression, "(default, default) == (default, default)").WithArguments("==", "default", "default").WithLocation(6, 30),
                 // (7,30): error CS0034: Operator '==' is ambiguous on operands of type 'default' and '(default, default)'
                 //         System.Console.Write(default == (default, default));
```

Comment:
This looks like the wrong error code.  Neither `default` nor `(default, default)` is a type.

Reply:
This hasn't been changed. 
I think it's related to the `Type` null test you were suggesting above, I'll add a PROTOTYPE marker.